recipes/plasmapy/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set version = "0.1.2" %}
+
+package:
+  name: plasmapy
+  version: {{ version }}
+
+source:
+  fn: plasmapy-{{version}}.tar.gz
+  url: https://pypi.io/packages/source/p/plasmapy/plasmapy-{{ version }}.tar.gz
+  sha256: 2a2bbc064a8b5726bc32447973babba9355e5c7d9483b1eebc82c6e83a9a173e
+
+build:
+  number: 0
+  script: python setup.py install --offline --single-version-externally-managed --record=record.txt
+  skip: True    # [py < 36]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+  host:
+    - python
+    - cython
+    - setuptools
+    - numpy
+    - scipy
+    - astropy
+  run:
+    - python
+    - numpy
+    - scipy
+    - astropy
+    - pytest
+    - mpmath
+    - h5py
+    - lmfit
+    - matplotlib
+    - colorama
+    - pytest-astropy
+
+test:
+  imports:
+    - plasmapy
+    - plasmapy.atomic
+    - plasmapy.atomic.tests
+    - plasmapy.classes
+    - plasmapy.classes.tests
+    - plasmapy.constants
+    - plasmapy.io
+    - plasmapy.mathematics
+    - plasmapy.mathematics.tests
+    - plasmapy.physics
+    - plasmapy.physics.tests
+    - plasmapy.utils
+    - plasmapy.utils.tests
+    - plasmapy.visualization
+    - plasmapy.visualization.tests
+
+about:
+  home: https://github.com/PlasmaPy/PlasmaPy
+  license: BSD-3-Clause
+  license_family: BSD
+  #license_file: LICENSE.md
+  summary: Python package for plasma physics
+  description: 'PlasmaPy is a community-developed and community-driven Python package for plasma physics.'
+  doc_url: 'http://plasmapy.readthedocs.io/en/docs/'
+  dev_url: 'https://github.com/plasmapy/plasmapy'
+
+extra:
+  recipe-maintainers:
+    - SolarDrew
```

Comment:
@SolarDrew , do you agree to be a maintainer as well?

Reply:
Sorry, just catching up on this. Yes, I'm happy to be a maintainer :)